### DISCUSSION TITLE
[Feature Request] Always show hidden items if monitor screen is larger than <enter resolution>

### DISCUSSION THREAD

**Discussion by jalurgemilang** [jordanbaird/Ice#294] [General]
I feel that this feature is very useful for users who have ultrawide monitors or, using multiple monitors of different sizes. In ultrawide monitors, the hidden items are always shown. On the secondary, smaller monitor, they are hidden.

**Comment by jordanbaird** [comment]
I've moved this over to issue #299. Let's have any further discussion there.
